src/object/cli_obj.c

```diff
@@ -3722,6 +3722,8 @@ obj_comp_cb(tse_task_t *task, void *data)
 				obj_ec_recov_data(&obj_auxi->reasb_req,
 					obj->cob_md.omd_id, args->nr);
 
+				args->iods = obj_auxi->reasb_req.orr_uiods;
+				args->sgls = obj_auxi->reasb_req.orr_usgls;
```

Comment:
is the purpose to reuse it when retry the IO?
1) seems no need to retry here
2) even if for retry, it will not reassemble again when retry the IO, so should keep it as reassembled iod/sgls, no?

Reply:
hmm, iods&sgls will be freed in the following free. so this is trying to restore the original value.  Retry is actually means the retry in dsc_obj_retry_cb() which might be called after obj_comp_cb(), and it might reuse. 
But I just realize that we do not need retry for server for now. Hmm, even so, I think arg should restore iods/sgls to its original value, in case other callback will reuse it?